components/src/core/info-list-item/info-list-item.component.ts

```diff
@@ -47,6 +47,13 @@ import { requireContent, isEmptyView } from '../../utils/utils';
             >
                 <ng-content select="[pxb-subtitle]"></ng-content>
             </div>
+            <div
+                class="mat-body-2 pxb-info-list-item-info-wrapper"
+                matLine
+                [class.pxb-info-list-item-wrap]="wrapSubtitle"
```

Comment:
This is the only thing that kinda stands out to me; do you think there would be benefit in introducing a new variable `wrapInfo`?  I really doubt this would get much use, but it does help with the flexibility of the component.

Reply:
I just used the wrapSubtitle, making the assumption that if you wanted the subtitle text to wrap then all of it should wrap(including the third line). We can certainly make that update if that's what we want to do. @joebochill and I were discussing the possibility of just having a wrapText prop that would make it an all or nothing kind of thing. Not sure which direction we want to go.

Reply:
We're also likely going to refactor this in an upcoming major release to be more like angular (take out the logic for doing the separator stuff and just allow any ReactNode as the subtitle).

Reply:
Okay that's fine with me, I don't think it's an issue.  This PR looks good to me!